src/server/index.ts

```diff
@@ -68,6 +68,13 @@ router.use(activityPub.routes());
 router.use(nodeinfo.routes());
 router.use(wellKnown.routes());
 
+router.get('/twemoji/:x', ctx => {
+	const file = fs.createReadStream(`${__dirname}/../../twemoji/assets/svg/${ctx.params.x}.svg`);
```

Comment:
ここ直接 fs 入るの心理的怖さがあるので、制約つけてもいいかも

```suggestion
router.get('/twemoji/:x([\\da-f]+)', ctx => {
	const file = fs.createReadStream(`${__dirname}/../../twemoji/assets/svg/${ctx.params.x}.svg`);
```

Reply:
ちなみに .svg 消した理由って特にあるかしら